stable_diffusion/stable_diffusion_tpu.py

```diff
@@ -0,0 +1,140 @@
+"""Ray Serve Stable Diffusion example."""
+from io import BytesIO  # pylint:disable=g-importing-member
+from typing import List
+from fastapi import FastAPI  # pylint:disable=g-importing-member
+from fastapi.responses import Response  # pylint:disable=g-importing-member
```

Comment:
Can these `pylint` commends be removed?

Reply:
Done in [a1d6ed0](https://github.com/ray-project/serve_config_examples/pull/8/commits/a1d6ed0d63ab3c291b88d18bae017c607dc8d02b).